.github/workflows/package-release.yml

```diff
@@ -91,20 +105,68 @@ jobs:
         env:
           BUILD_DIR: ${{ steps.build-driver.outputs.build-dir }}
 
+      - name: "Cache build artifacts for subsequent builds"
+        uses: actions/cache/save@v4
+        with:
+          key: ${{ github.sha }}-${{ matrix.php }}-${{ matrix.ts }}-${{ matrix.arch }}
+          enableCrossOsArchive: true
```

Comment:
This is the notable difference to the same step in the windows-tests workflow. This took me longer to figure out than I care to admit.

Reply:
Is this why `windows-latest` is required above? If so, I'd suggest adding a comment there to point out the relationship to `enableCrossOsArchive`.

Context: https://github.com/actions/cache/blob/main/tips-and-workarounds.md#cross-os-cache

Reply:
Added a comment in both jobs to explain this.